packages/math/triangle.ts

```diff
@@ -0,0 +1,47 @@
+import type { GlobalPoint, LocalPoint } from "./types";
+
+// Types
+
+/**
+ * A triangle represented by 3 points
+ */
+export type Triangle<P extends GlobalPoint | LocalPoint> = [
+  a: P,
+  b: P,
+  c: P,
+] & {
+  _brand: "excalimath__triangle";
+};
+
+/**
+ * Tests if a point lies inside a triangle. This function
+ * will return FALSE if the point lies exactly on the sides
+ * of the triangle.
+ *
+ * @param triangle The triangle to test the point for
+ * @param p The point to test whether is in the triangle
+ * @returns TRUE if the point is inside of the triangle
+ */
+export function triangleIncludesPoint<P extends GlobalPoint | LocalPoint>(
```

Comment:
if we want to put `triangle` in front, then we should probably follow the same convention in other files

Reply:
What do you mean?

Reply:
for instance, in polygon.ts, we have `pointInPolygon`, so either we change this to `pointInTriangle` or change `pointInPolygon` to `polygonIncludesPoint`